int64 clID
string clSlot
string dsID
---
string result
